com.unity.uiwidgets/Runtime/cupertino/date_picker.cs

```diff
@@ -927,6 +937,9 @@ class _CupertinoDatePickerDateState : State<CupertinoDatePicker> {
                     backgroundColor: widget.backgroundColor,
                     squeeze: CupertinoDatePickerUtils._kSqueeze,
                     onSelectedItemChanged: (int index) => {
+                        if (index < 0) {
```

Comment:
is this ok? this index looks like the number of day in a month. If it is, it is necessary to add index with 31/30/28 accordingly instead of 31 for all months?